src/man/sssd-ad.5.xml

```diff
@@ -1081,17 +1081,38 @@ ad_gpo_map_deny = +my_pam_service
                         <para>
                             This option should only be used to test the machine
                             account renewal task. The option expects 3 integers
-                            separated by a colon (':'). The first integer
-                            defines the interval in seconds how often the task
-                            is run. The second specifies the initial timeout in
-                            seconds before the task is run for the first time
-                            after startup. The optional third value specifies a
-                            maximal random offset to the previous two values to
-                            avoid updates of many hosts at the same time
-                            ("thundering herd problem").
-                        </para>
-                        <para>
-                            Default: 86400:750:300 (24h, 12m30s and 5m)
+                            and a string separated by a colon (':'). The first
+                            integer defines the interval in seconds how often
+                            the task is run. The second specifies the initial
+                            timeout in seconds before the task is run for the
+                            first time after startup. The optional third value
+                            specifies a maximal random offset to the previous
+                            two values to avoid updates of many hosts at the
+                            same time ("thundering herd problem").
+                        </para>
+                        <para>
+                            The optional fourth string value identifies the
```

Comment:
As a user, having read that both third and fourth values are optional, I wouldn't be sure if I can skip third but specify fourth. Like `86400:750::realm`?

Reply:
I added to the man page and to the code handling those cases for the third and fourth value.